addons/survey/static/src/js/survey_renderers.js

```diff
@@ -0,0 +1,52 @@
+/** @odoo-module */
+
+import Dialog from 'web.Dialog';
+import KanbanRenderer from 'web.KanbanRenderer';
+import ListRenderer from 'web.ListRenderer';
+import { qweb as QWeb, _t } from 'web.core';
+
+const SurveySampleModalMixin = {
+    _onOpenModal: function () {
```

Comment:
"_onOpenModal" is a bit too generic, let's make sure we don't conflict with other methods by using something like "_onOpenSurveySampleModalClick".

Also: might want to add a bit of docstring to explain what we are doing here.